tests/test_density_RDF.py

```diff
@@ -81,14 +78,9 @@ def test_random_point(self):
         tolerance = 0.1
         box_size = r_max*3.1
 
-        def ig_sphere(x, y, j):
-            return 4/3*np.pi*np.trapz(y[:j]*(x[:j]+dr/2)**2, x[:j])
-
         for i, r_min in enumerate([0, 0.05, 0.1, 1.0, 3.0]):
             nbins = int((r_max - r_min) / dr)
             box, points = util.make_box_and_random_points(box_size, num_points)
-            points.flags['WRITEABLE'] = False
```

Comment:
We really need to test read-only arrays somewhere else in freud if not here. Let's edit some system setup functions in `util.py` to generate and use read-only NumPy arrays.

Reply:
Yes, I'll make an issue for adding proper tests for this. I don't think sprinkling it in randomly in other tests is the best idea, we need a more systematic approach.